server/@types/supportAdditionalNeedsApi/index.d.ts

```diff
@@ -416,6 +432,11 @@ export interface components {
        * @example An LNSP is required for approx 1 hour per week to read and explain passages of text to Chris.
        */
       lnspSupport?: string
+      /**
+       * @description Optional Additional information about the plan that is not covered in the other questions.
+       * @example Chris is very open about his issues and is a pleasure to talk to.
+       */
+      detail?: string
```

Comment:
Reimporting the swagger spec has brought in a few new things from the API (like the new reference data endpoint ☝️ ), but in context of this PR the thing we are interested in is the new `detail` field in the request and response objects 👍